rapids-cmake/cpm/versions.json

```diff
@@ -9,14 +9,7 @@
       "version": "2.5.0",
       "git_shallow": false,
       "git_url": "https://github.com/NVIDIA/cccl.git",
```

Comment:
Since you dropped the patch for cccl, you need to update the `cpm_find-patch-command` test to validate against a different project we patch

Reply:
I looked at `versions.json` but I don't see any projects with long-term patches. We have patches for Thrust/libcudacxx but those will be going away. We also have patches for fmt and spdlog but those may be updated soon.

I can change this to libcudacxx but it's just kicking the can down the road.

Reply:
@robertmaynard Please take a look now. I updated this in 711f4b0.